pipenv/core.py

```diff
@@ -1574,6 +1548,166 @@ def pip_install(
     return c
 
 
+def pip_install_deps(
```

Comment:
This started as a copy of `pip_install` and as modified to take a list of dependencies instead of a single package.